vendor.conf

```diff
@@ -80,7 +80,7 @@ google.golang.org/grpc v1.12.0
 # This commit does not need to match RUNC_COMMIT as it is used for helper
 # packages but should be newer or equal.
 github.com/opencontainers/runc 96ec2177ae841256168fcf76954f7177af9446eb
-github.com/opencontainers/runtime-spec 5684b8af48c1ac3b1451fa499724e30e3c20a294 # v1.0.1-49-g5684b8a
+github.com/opencontainers/runtime-spec 29686dbc5559d93fb1ef402eeda3e35c38d75af4 # v1.0.1-59-g29686db
```

Comment:
@jhowardmsft do you anticipate more changes to come for Windows, or should we ask the runtime-spec maintainers to consider tagging a new release?

/cc @tianon